src/exchangeRates.js

```diff
@@ -45,31 +49,38 @@ async function get(params) {
   const today= new Date().toISOString().split('T')[0];
   const {fromCurrency='AUD', toCurrency='USD', onDate=today} = params;
   let exchangeRates = await db.query(
-    `SELECT rate, created_at FROM exchange_rates WHERE from_currency = ? AND to_currency = ? AND on_date = ?`, 
+    `SELECT rate, created_at FROM exchange_rates WHERE from_currency = ? AND to_currency = ? AND on_date = ?`,
     [fromCurrency, toCurrency, onDate]
   );
   if (exchangeRates.length) {
     const rate = Number(exchangeRates[0].rate);
     console.log(`Found exchange rate of ${rate} for ${fromCurrency} to ${toCurrency} of ${onDate} in the db`);
-    
+
     return {fromCurrency, toCurrency, onDate, rate};
   }
-  
+
   return getExternal(fromCurrency, toCurrency, onDate);
 }
 
-async function getMultiple() {
+async function getMultiple(currentPage) {
+  let itemsPerPage = 10;
```

Comment:
can you please put this in config.js and pull it from there.

Reply:
Do you mean for itemsPerPage only?

Reply:
Yes